gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/src/AuthorizationService.js

```diff
@@ -100,125 +100,510 @@ var fluid = fluid || require("infusion");
                 func: "{dataStore}.revokeClientCredentialsToken"
                     // clientCredentialsTokenId
             }
+        },
+        events: {
+            onGrantAuthorizationCode: null,
```

Comment:
Add docs here to explain that these are pseudoevents rather than standard events